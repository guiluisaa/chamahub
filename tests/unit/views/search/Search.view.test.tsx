import React from 'react';
import { shallow } from 'enzyme';

import SearchView from '@/views/search/Search.view';

jest.mock('@/io/user/useUser.hook', () => () => ({
  user: {},
  getUser: jest.fn(),
  isLoading: false,
}));

jest.mock('react-router-dom', () => ({
  useHistory: () => ({ replace: jest.fn() }),
}));

describe('<SearchView />', () => {
  it('should match snapchat', () => {
    const wrapper = shallow(<SearchView />);
    expect(wrapper).toMatchSnapshot();
  });
});
