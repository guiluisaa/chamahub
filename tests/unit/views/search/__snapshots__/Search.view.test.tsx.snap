// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchView /> should match snapchat 1`] = `
<Styled(styled.div)>
  <SearchForm
    isLoading={false}
    onSubmit={[Function]}
  />
  <br />
  <UserCard
    user={Object {}}
  />
</Styled(styled.div)>
`;
